### DISCUSSION TITLE
Classroom TAs invitation link leads to 404

### DISCUSSION THREAD

**Discussion by MeckeJadeHS** [community/community#24155] [GitHub Education]
<p>Hi –</p>
<p>Regarding GitHub classroom (I am a huge fan already)… I created a classroom, created assignments and so forth – everything fine so far.</p>
<p>Now I want to invite my TAs, who should take over the task. They are already part of the organization. The created invite link sends them to 404.</p>
<p>Since the design of the process is straight forward, I don’t think I am doing something wrong and suspect a bug. Please let me know if I am wrong. However, we keep trying. For details, please get in touch with me.<br>
The bug report categories did not work for me - I did not know where to put classroom topics.</p>
<p>Best regards &amp; Thanks for the support</p>
<p>Kai</p>

**Comment by MeckeJadeHS** [comment, marked as answer]
<p>Found it…<br>
The TAs must be “owners” in the “organisation”.</p>
<aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/education/classroom/issues/1566" target="_blank" rel="noopener nofollow ugc">github.com/education/classroom</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/education/classroom/issues/1566" target="_blank" rel="noopener nofollow ugc">Link for admin invitation ends with 404</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2018-09-14" data-time="09:19:01" data-timezone="UTC">09:19AM - 14 Sep 18 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/rguillome" target="_blank" rel="noopener nofollow ugc">
          <img alt="rguillome" src="https://user-images.githubusercontent.com/54799287/181083069-98af0a1c-632d-46ac-9d2d-c818de56482e.png" class="onebox-avatar-inline" width="20" height="20">
          rguillome
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">Issue
What you were trying to do ?
I trying to join a classroom as an admin from an invitation link
What happened (include...</p>
</div>

<div class="labels">
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">bug :beetle:</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">stale</span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Dimas-commits-oss** [comment]
<p>Uvjcpbidghcouobkcorjbrigienwknd</p>
